### PR TITLE
Discover memories on this day

### LINKED ISSUE #720: Discover memories like today x years ago
A memories feature like facebook, on this day x amount of years ago these are your memories? Piwigo has a feature like this. Face detection is moving forward on photoprism quicker than piwigo i think. Synology moments does face detection but seemed laggy when i used it last and it needed its own folder order i believe.
thanks

### PR DESCRIPTION
**Pull request by kvalev**
closes #720

### CODE REVIEW

**Comment by kvalev** [comment]
There are few things missing and it might be a bit buggy, but I am happy to get some feedback.

**Comment by kvalev** [comment]
Tested and added few more things and I think the PR is now ready for testing and code review.

**Comment by lastzero** [comment]
Thanks! Looks like you've reactivated our (hidden) discover page? Didn't have time to work this out for our stable release - and most users also didn't seem to care much about it. Basically no feedback while it was enabled in our preview last year.

Happy to give it a try once we have time, going to discuss this with @graciousgrey :+1:

File: frontend/src/component/photo/cards.vue [hunk 1]
```diff
@@ -90,7 +90,7 @@
               <v-icon color="white" class="select-on">lock</v-icon>
             </v-btn>
 
-            <v-btn :ripple="false"
+            <v-btn v-if="selectMode" :ripple="false"
```
Comment (lastzero) [inline frontend/src/component/photo/cards.vue:93]: If you needed to add this: Is there a bug in our stable release at the moment? Visibility should be set via CSS already.

Comment (kvalev) [reply frontend/src/component/photo/cards.vue:93]: Yes, I think there is a bug - the circle was being shown in the new UI, although I have disabled the selection. There might be other ways to hide it as well, but this check is needed IMO.

**Comment by kvalev** [comment]
> Thanks! Looks like you've reactivated our (hidden) discover page? Didn't have time to work this out for our stable release - and most users also didn't seem to care much about it. Basically no feedback while it was enabled in our preview last year.
> 
> Happy to give it a try once we have time, going to discuss this with @graciousgrey 👍

Well the page is still hidden, so nothing has changed in that regard. The discovery options in the current version might be a bit exotic/uncommon (colors, season etc). The most prominent ways to discover photos IMO is either `on this day` or `random` (based on some criterias). The others can be discarded I think.

File: frontend/src/component/photo/cards.vue (hunk 1 above)
Comment (lastzero) [reply frontend/src/component/photo/cards.vue:93]: How you can disable selecting pictures completely? AFAIK you can always select pictures...

Comment (kvalev) [reply frontend/src/component/photo/cards.vue:93]: `selectMode` does exactly that - disable selecting pictures.

Comment (lastzero) [reply frontend/src/component/photo/cards.vue:93]: We've used it in a positive sense to always show the button when at least one item is selected. You should always see it on hover / mouse over, even if no item is selected.

**Comment by JackJones2018** [comment]
Hi Guys,

Is this now included in the current version? I've just downloaded the latest docker image from photoprism/photoprism:latest
210523-b1856b9d-Linux-x86_64.

Which option on the left hand side shows the memories on this day?
Also is there an android app that will show these notifications?
thanks

**Comment by graciousgrey** [comment]
@JackJones2018 it is not yet included. We need to finish high priority tasks like face recognition and user management first. Just follow the progress of this PR once it is merged it will be part of the next release :)

**Comment by RogerSik** [comment]
Why closed? Was looking forward to this. 😥

**Comment by jccroft1** [comment]
> Why closed? Was looking forward to this. 😥

Same 👍

**Comment by graciousgrey** [comment]
Don't worry this feature is still planned see: https://github.com/photoprism/photoprism/issues/720 & https://github.com/photoprism/photoprism/issues/153.

We need to finish high priority tasks like facial recognition, multi user and batch edit first. We are already working more than full time on this project and testing, and adapting pull requests takes more time than most people would expect.

**Pull request closed without merging**